tools/p4mlir-export/json_emitter.h

```diff
@@ -0,0 +1,41 @@
+/* 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+*/
+
+#ifndef P4MLIR_JSON_EMITTER_H
+#define P4MLIR_JSON_EMITTER_H
+
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/Operation.h"
+#include "llvm/Support/JSON.h"
+
+namespace P4::P4MLIR {
+
+class JsonEmitter {
+public:
+
+    // Emit a P4HIR-ISO json, given MLIR module.
```

Comment:
As explained above, let's not invent this P4HIR-ISO json format here. Instead, try to emit the BMv2 JSON format directly.

I'd suggest that you only keep the `emitModule` function, and start with returning an empty or constant JSON value. Once you have that working (in the sense that the added test passes, see another comment), we can discuss the next steps.